### PR TITLE
New version: BlenderFoundation.Blender version 4.0.1

### PR DESCRIPTION
**Pull request by PckgrBot**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/127219)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
It seem the server did not return file size.  This link https://www.blender.org/download/release/Blender4.0/blender-4.0.1-windows-x64.msi is a webpage.  The Sigcheck did not detect as publisher.

File: manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- fbx
+- glb
+- gltf
+- obj
+- ply
+- stl
+ProductCode: '{B557A37F-811D-4A98-9306-FB34E069D999}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.blender.org/download/release/Blender4.0/blender-4.0.1-windows-x64.msi
```
Comment (BrandonWanHuanSheng) [inline manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml:24]:
```suggestion
  InstallerUrl:  https://ftp.halifax.rwth-aachen.de/blender/release/Blender4.0/blender-4.0.1-windows-x64.msi
```

**Review by BrandonWanHuanSheng — CHANGES_REQUESTED** [review]
The link direct to the binary file is https://ftp.halifax.rwth-aachen.de/blender/release/Blender4.0/blender-4.0.1-windows-x64.msi

File: manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ FileExtensions:
 ProductCode: '{B557A37F-811D-4A98-9306-FB34E069D999}'
 Installers:
 - Architecture: x64
-  InstallerUrl: https://www.blender.org/download/release/Blender4.0/blender-4.0.1-windows-x64.msi
+  InstallerUrl:  https://ftp.halifax.rwth-aachen.de/blender/release/Blender4.0/blender-4.0.1-windows-x64.msi
```
Comment (Trenly) [inline manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml:24]: This points to a specific mirror, which isn’t preferred. Try using the same format as previous versions - 
```suggestion
https://download.blender.org/release/Blender4.0/blender-4.0.1-windows-x64.msi
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by BrandonWanHuanSheng — APPROVED** [review]
It seem you were online

File: manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml]: Thats a specific mirror, which won’t be the best URL for all users. Its best to point to a CDN wherever possible, just like previous versions of this package do

**Review by Trenly — APPROVED** [review]

**Comment by BrandonWanHuanSheng** [comment]
Is there is more than one mirror?

**Comment by BrandonWanHuanSheng** [comment]
@PckgrBot where is your InstallerUrl

**Comment by Trenly** [comment]
> Is there is more than one mirror?

Most companies use a content distribution network (CDN) which is location based. Most users will only ever be aware of one mirror because their location will only allow them to connect to one. I get a different mirror when I connect.

In general, the direct URL is better, but that only really applies if there is only one download location and no CDN is being used.

IMG_1763

File: manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- fbx
+- glb
+- gltf
+- obj
+- ply
+- stl
+ProductCode: '{B557A37F-811D-4A98-9306-FB34E069D999}'
+Installers:
+- Architecture: x64
+https://download.blender.org/release/Blender4.0/blender-4.0.1-windows-x64.msi
```
Comment (BrandonWanHuanSheng) [inline manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml:24]:
```suggestion
 InstallerUrl: https://download.blender.org/release/Blender4.0/blender-4.0.1-windows-x64.msi
```

**Comment by BrandonWanHuanSheng** [comment]
The InstallerURL was missing on the manifest

**Review by Trenly — APPROVED** [review]
I need more coffee...

File: manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- fbx
+- glb
+- gltf
+- obj
+- ply
+- stl
+ProductCode: '{B557A37F-811D-4A98-9306-FB34E069D999}'
+Installers:
+- Architecture: x64
+ InstallerUrl: https://download.blender.org/release/Blender4.0/blender-4.0.1-windows-x64.msi
```
Comment (BrandonWanHuanSheng) [inline manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml:24]:
```suggestion
  InstallerUrl: https://download.blender.org/release/Blender4.0/blender-4.0.1-windows-x64.msi
```

**Comment by BrandonWanHuanSheng** [comment]
@PckgrBot

**Review by Trenly — APPROVED** [review]

**Pull request merged**